src/backend/pylintrc

```diff
@@ -25,7 +25,7 @@ jobs=0
 load-plugins=pylint_django,pylint.extensions.no_self_use
 
 # Pickle collected data for later comparisons.
-persistent=yes
+persistent=no
```

Comment:
Is it voluntary to change this ?

Reply:
Yes it is, otherwise pylint try to save files on disk to be able to compare the current lint with the previous one. And this is useless for us, the container is deleted once the lint ended. So the file is lost.